src/main.cpp

```diff
@@ -280,6 +284,9 @@ int main(int argc, char** argv)
             case 'n':
                 filedelete_use_injector = true;
                 break;
+            case 'X':
```

Comment:
Lower-case `x` is already used as a flag so this is not ideal, how about `b`?

Reply:
Agreed